testdata/storage_export/buckets.json

```diff
@@ -0,0 +1,7 @@
+{
+  "buckets": [
+    {
+      "id": "codelabz-966e5.appspot.com"
```

Comment:
is this your personal bucket configuration?

Reply:
> is this your personal bucket configuration?

yes

Reply:
so is it okay to others to use it?

Reply:
> so is it okay to others to use it?

yes sir,firebase bucket files are used to share data only I guess

Reply:
ah sure

Reply:
> so is it okay to others to use it?

and more over it is storage emulator's meta data like profile photo so may be a problem.